mcstatus/tests/test_server.py

```diff
@@ -116,11 +117,13 @@ def test_status(self):
 
         assert self.socket.flush() == bytearray.fromhex("0F002F096C6F63616C686F737463DD01010009010000000001C54246")
         assert self.socket.remaining() == 0, "Data is pending to be read, but should be empty"
-        assert info.raw == {
-            "description": "A Minecraft Server",
-            "players": {"max": 20, "online": 0},
-            "version": {"name": "1.8", "protocol": 47},
-        }
+        assert info == JavaStatusResponse(
+            players=JavaStatusPlayers(max=20, online=0, list=[]),
+            version=JavaStatusVersion(name="1.8", protocol=47),
+            motd="A Minecraft Server",
+            latency=info.latency,
+            icon=None,
+        )
```

Comment:
I'm not sure should I change behavior same to dda07c207c9acc3ebcab2755bdbbfa9bfc91f323 or not. Because there isn't easy change behavior with monkeypatch, and I don't know how better change it.

Reply:
No, since this is testing the server's ability to obtain correct data from status, not `status` method itself, so it makes sense to verify that the returned `info` here is a valid `JavaStatusResponse` with the same values.